Backend/NUnit_Collections/NUnit_Collection_Tests/Collections.cs

```diff
@@ -124,12 +133,21 @@ public void Test_Collection_GetByIndex()
             Assert.AreEqual(expected, actual);
         }
 
+        [Test]
```

Comment:
Try to make this test using [TestCase("Whte")] and [TestCase("Black")]

Reply:
if you've looked at summator project you'll see my tests with TestCase like Dimo has told us on the lecture. :)